Makefile

```diff
@@ -20,7 +23,9 @@ test-ci:
 	@NODE_ENV=test NODE_PATH=lib XUNIT_FILE=$(CIRCLE_TEST_REPORTS)/mocha.xml \
 	./node_modules/.bin/istanbul cover \
 	./node_modules/mocha/bin/_mocha --report lcovonly -- -R xunit-file
+ifdef CODECLIMATE_REPO_TOKEN
```

Comment:
Ok, I seem to remember struggling with some issue in CircleCI and so ran the `test-ci` locally. I'll look at cleaning this up and seeing if `test-cov` is running fine locally and just remove this #ifdef and the other `CIRCLE` vars.